### ISSUE TITLE
Can no longer support this repo

### ISSUE THREAD

**Issue by tammyhart** [tammyhart/Reusable-Custom-WordPress-Meta-Boxes#41]
I am looking for someone who might be willing to take over support for this. I no longer use this library for my own work, and thus cannot continue to support it. If you're interested, leave a comment here.

**Comment by wesdekoninck** [comment]
If you don't mind my asking, are you using a different setup for creating custom meta boxes dynamically in Wordpress that you would recommend in place of what you have created here?

**Comment by tammyhart** [comment]
Yes, I just create them each individually by scratch. In the end, it easier for a team to support together, easier to debug and better for making sure proper data validation and sanitation is happening.

**Comment by wesdekoninck** [comment]
Makes sense! Thanks for your input and for this plugin, it has made several of my client projects easier.  I really appreciate the time you put in to it.

**Comment by vinoddalvi** [comment]
@tammyhart : I am interested to support it in my spare time.

**Comment by tammyhart** [comment]
Hi Vinod, I'd welcome it. Do you know if I can turn the repo over to someone or if you should just make a fork? I'm not that familiar with all of this.

**Comment by vinoddalvi** [comment]
Hi Tammy, Thank you very much.
You will find information about how to transfer the repository on the following page.

https://help.github.com/articles/how-to-transfer-a-repository

Cheers

**Comment by jmfortunatojr** [comment]
Hi Tammy. Thanks for the great repo.

I use your metaboxes on all of my projects, and recently went through and fixed most of the issues that the code has since you stopped supporting it. Mostly clean up of deprecated jQuery functions, as well as a few syntax changes with the WP media uploader. My version fixes the issues with repeatable images, file uploader not working, and a few others. Interested in merging to your master branch?
